scientist-softserv/palni-palci
# 🎁 Allowing per tenant ability to disable IIIF Print

The current (as of <2023-08-10 Thu>) implementation of IIIF Print assumes that we generate derivatives at the application level.  However, we want options for generating at the tenant level.

As noted, I have chosen to default using IIIF print to `true` because that was the behavior before this commit.

In other words, this commit is functionally a non-change to the production code-base (assuming no one goes and flips some switches).

Related to:

- https://github.com/scientist-softserv/palni-palci/issues/656
- https://github.com/scientist-softserv/palni-palci/issues/657
- https://github.com/scientist-softserv/palni-palci/issues/658
- https://github.com/scientist-softserv/palni-palci/issues/659

# Expected Behavior Before Changes

We always, regardless of configuration, use IIIF Print and therefore split PDFs and extract OCR.

# Expected Behavior After Changes

Unless a tenant goes and turns off "use_iiif_print" there are no functional changes.

<details>
<summary>Screenshot of Flipflop dashboard with new feature</summary>
<img width="1179" alt="Screenshot of IIIF Print as flippable feature" src="https://github.com/scientist-softserv/palni-palci/assets/2130/7327775c-d720-47c3-8ba1-41b54860821b">
</details>

# Verification

When they do disable "use_iiif_print" the way to verify that we're not splitting PDFs is to:

1. Create a new work with a PDF
2. Check that the work does not have child image objects attached (e.g. pages extracted from the PDF)
